### DISCUSSION TITLE
Pi-hole in Docker in PROXMOX installiert Fehlermeldung: Disk shortage

### DISCUSSION THREAD

**Discussion by themanfrommoon** [RPiList/specials#1446] [Q&A] [answered]
Moin,

ich habe pihole mit folgender "docker-compose.yaml" installiert:
`version: "3.7"

services:
  pihole:
    image: pihole/pihole:latest
    container_name: 07_Pihole
    # For DHCP it is recommended to remove these ports and instead add: network_mode: "host"
    restart: unless-stopped    
    environment:
      TZ: Europe/Berlin
      WEBPASSWORD: raspi
      DNS1: 1.1.1.1
      DNS2: 1.0.0.1
    ports:
      - "53:53/tcp"
      - "53:53/udp"
      # DHCP Server Usage
      #- "67:67/udp"
      - "80:80/tcp"
      - "443:443/tcp"     
    # Volumes store your data between container upgrades
    volumes:
      #- ./data/pihole/conf/:/etc/pihole/
      #- ./data/dnsmasq.d/conf/:/etc/dnsmasq.d/
      - 07_Pihole_pihole:/etc/pihole/
      - 07_Pihole_dsnmasq.d:/etc/dnsmasq.d/
      # run `touch ./var-log/pihole.log` first unless you like errors
      # - './var-log/pihole.log:/var/log/pihole.log'
    dns:
      - 127.0.0.1
      - 1.1.1.1
    labels:
        com.centurylinklabs.watchtower.enable: "true"
    # Recommended but not required (DHCP needs NET_ADMIN)
    #   https://github.com/pi-hole/docker-pi-hole#note-on-capabilities
    #cap_add:
    #  - NET_ADMIN
    networks:
      - 07_Pihole_net

volumes:
  07_Pihole_pihole:
    name: "07_Pihole_pihole"
  07_Pihole_dsnmasq.d:
    name: "07_Pihole_dsnmasq.d"

networks:
  07_Pihole_net:
    name: "07_Pihole_net"
    internal: false
    driver: bridge
    driver_opts:
      com.docker.network.bridge.name: br-pihole
`

Leider führt das am Ende dazu, dass unter 'Pi-hole diagnosis' zwei Fehler angezeigt werden:
![grafik](https://github.com/RPiList/specials/assets/28815063/7e1a1079-43b3-48cf-956f-0f2040d6df03)
Offenbar ist irgendein Speicherplatz zu klein. Soweit so klar.

Das pihole läuft in Docker.
Docker läuft in einem LXC Container in PROXMOX auf einer gejailbreakted ASUS CHROMEBOX 3 Intel i7-8550U 16GB 64GB SSD.
Specherplatz müsste also genug da sein.
Ich nehme an, dass irgendwo nur eine Einstellung nicht passt.

Die Frage ist jetzt nur: Wo/Wie behebe ich das?

Vielen Dank und beste Grüße,
Chris

**Comment by LizenzFass78851** [comment, marked as answer]
Vermutlich wurde den LXC Container nicht genügend Speicherplatz zugewiesen.

Versuche den LXC Container mehr Speicherplatz zuzuweisen.

Anleitung:
https://cohob.de/proxmox-ct-volume-vergroessern

**Reply by themanfrommoon** [reply]
Wunderbar, ich bedanke mich!
Mit dem Tipp konnte ich das Problem lösen.
Nun scheint es zu funktionieren!
Besten Dank, Chris
